lstchain/reco/r0_to_dl1.py

```diff
@@ -338,8 +338,8 @@ def r0_to_dl1(
                                            new_ped=True, new_ff=True)
 
                 # flat-field or pedestal:
-                if ( event.trigger.event_type == EventType.FLATFIELD or
-                     event.trigger.event_type == EventType.SKY_PEDESTAL ):
+                if (event.trigger.event_type == EventType.FLATFIELD or
+                        event.trigger.event_type == EventType.SKY_PEDESTAL):
```

Comment:
Overindent?

Reply:
Is it?
I don't know all the rules, IDE auto-formatting.

Reply:
black would format as

```
if (
    event.trigger.event_type == EventType.FLATFIELD
    or event.trigger.event_type == EventType.SKY_PEDESTAL
):
```

Reply:
Thanks, I'll see how to add black to PyCharm.